components/inputs/documentation/date-time-inputs.md

```diff
@@ -0,0 +1,109 @@
+# Date & Time Inputs
+
+## Date Inputs
+
+The `<d2l-input-date>` component consists of a text input field for typing a date and an attached calendar (`<d2l-calendar>`) dropdown. The dropdown opens on click of the text input, or on enter or down arrow press if the text input is focused. It displays the `value` if one is specified, or a placeholder if not, and reflects the selected value when one is selected in the `calendar` or entered in the text input.
+
+![example screenshot of date input](../screenshots/date.gif?raw=true)
+
+```html
+<script type="module">
+  import '@brightspace-ui/core/components/inputs/input-date.js';
+</script>
+<d2l-input-date
+  label="Start Date"
+  value="2020-11-20">
+</d2l-input-date>
+```
+
+**Properties:**
+
+- `label` (String, required): accessible label for the input
+- `disabled` (Boolean): disables the input
+- `label-hidden` (Boolean): hides the label visually (moves it to the input's `aria-label` attribute)
+- `value` (String, default: `''`): value of the input. This should be in ISO 8601 calendar date format (`YYYY-MM-DD`) and should be localized to the user's timezone (if applicable).
+
+**Accessibility:**
+
+To make your usage of `d2l-input-date` accessible, use the following properties when applicable:
+
+| Attribute | Description |
+|--|--|
+| `label` | **REQUIRED** [Acts as a primary label on the input](https://www.w3.org/WAI/tutorials/forms/labels/). Visible unless `label-hidden` is also used. |
+| `label-hidden` | Use if label should be visually hidden but available for screen reader users |
+
+**Events:**
+
+* `d2l-input-date-change`: dispatched when a date is selected or typed. `value` reflects the selected value and is in ISO 8601 calendar date format (`YYYY-MM-DD`).
```

Comment:
Noticed this as a discrepancy between the date and time inputs -- the name of the change event. Could we just go with `change` for the name here? Then it's consistent with `d2l-input-text`, `d2l-input-checkbox` and `d2l-input-time`. Would need to change `d2l-input-date-time` as well.

Reply:
Is there any use-case for knowing if it was the date or time specifically that changed in a date-time input, or are we assuming the complete value needs to be checked on any adjustment?